db/versionHistory.go

```diff
@@ -71,5 +71,6 @@ func init() {
 		{Major: 2, Minor: 3, Patch: 1},
 		{Major: 2, Minor: 3, Patch: 2},
 		{Major: 2, Minor: 4},
+		{Major: 2, Minor: 4, Patch: 2},
```

Comment:
Based on semver this is not a patch release as it adds a new feature. It will be in 2.5.0 so please update your migrations to this value here and in the migrations file